src/main/java/edu/duke/cs/osprey/gpu/MemoryBuffer.java

```diff
@@ -304,6 +306,6 @@ public byte getByte(Structs.Array f, long i) {
 
 	@Override
 	public void close() throws IllegalStateException {
-		base.close();
+		sharedScope.close();
```

Comment:
I'm kind of surprised this API keeps changing so much. I've never seem them incubate things for so long before.
As long as the tests pass, these updates are probably fine.

Reply:
Luckily changes weren't as large as last time, and I think this particular change makes sense.